src/utils/ReduxUtils.ts

```diff
@@ -2,8 +2,8 @@ import * as ReactRedux from 'react-redux';
 import * as Redux from 'redux';
 import {extend} from 'underscore';
 
-export type IDispatch = (action: IReduxAction<any> | IThunkAction) => void;
-export type IThunkAction = (dispatch: IDispatch, getState?: () => any) => void;
+export type IDispatch = (action: IReduxAction<any> | IThunkAction) => any;
+export type IThunkAction = (dispatch: IDispatch, getState?: () => any) => any;

```

Comment:
I think it would be more flexible to do: 
```typescript
export type IDispatch<T = void> = (action: IReduxAction<any> | IThunkAction) => T;
export type IThunkAction<T = void> = (dispatch: IDispatch, getState?: () => any) => T;
````